index.rst

```diff
@@ -74,17 +74,15 @@ exact contents of Root should depend on the type of Storage being initialized.
 The only current example of Root is in posix storage: Root is the path to the
 top folder of the Repository.
 
-Parent and Self
-^^^^^^^^^^^^^^^
+Parents
+^^^^^^^
 
-Repositories can point to other repositories as inputs. This relationship is
-defined in the repository configuration.
+Repositories may have none or more parents that are other repositories. These
+repositories are used by the butler as inputs. Repositories' parent
+configuration are persisted along with the repository's configuration.
```

Comment:
I'm not sure this is a good idea in the long term.  We might have tools that modify a repository and its configuration; "downstream" repositories that refer to it should not care.